swap_meet/decor.py

```diff
@@ -1,2 +1,20 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+
+    def __init__(self, id=None, width=0, length=0, condition=0 ):
+
+        super().__init__(id, condition)
+        
+        self.width = width
+        self.length = length
+        
+
+    def get_category(self):
+        return "Decor"
```

Comment:
While this works, we do end up with a get_category function in each class. While this isn't necessarily an issue, it could definitely qualify as a function perfect for inheritance. If we keep the get_category method in just the Item class, it can get passed down to its three children. We do have a way to return a class's name as well. If we did something like what you see below, the get_category function will return the class name without having to hardcode each category:

     def get_category(self):
          return self.__class__.__name__